numpy/add_newdocs.py

```diff
@@ -5463,18 +5442,22 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Parameters
     ----------
-    X : array_like
-        First input array.
-    Y : array_like
-        Second input array.
-    out : array_like
-        An array to store the output. Must be the same shape as the
-        output would have.
+    *x : array_like
```

Comment:
I agree that having numpydoc escape the `*`'s when generating the sphinx input would be nice, but it doesn't at this time. If this never showed up in the documentation, that would not be a problem, but I assume you would like it to eventually show up.